app/components/provider_interface/application_timeline_component.rb

```diff
@@ -32,6 +32,7 @@ def status_change_events
           title_for(change),
           actor_for(change),
           change.changed_at,
+          link_params_for_status(change),
```

Comment:
I liked `link_name` and `link_path`! Appreciate they're easier to represent in arrays. Could these be splatted here maybe?

Reply:
Addressed.